src/name/admitriev/jhelper/ui/ConfigurationDialog.java

```diff
@@ -40,6 +41,11 @@ public ConfigurationDialog(@NotNull Project project, Configurator.State configur
 				configuration.getRunFile(),
 				RelativeFileChooserDescriptor.fileChooser(project.getBaseDir())
 		);
+		archiveDirectory = new FileSelector(
```

Comment:
Maybe it makes sense to store/find absolute path here?
Archived files do not really "participate" in the project anyway, and it'll allow less concatentation back and forth. (and it'll require less changes when they finally remove project.getBaseDir())

On the other hand it will make it less consistent and also make it harder to move th project across the FS.

What do you think?

Reply:
Interesting, perhaps we can try to support both options by giving a special configuration item for the archive directory? It could look somehow like CardLayout component in swing. I can try to add this

Reply:
Not sure it's worth to support 2 options.
Do you know how CLion/IDEA handle paths themselves?